engines/dreamweb/titles.cpp

```diff
@@ -99,6 +101,11 @@ void DreamWebEngine::gettingShot() {
 }
 
 void DreamWebEngine::bibleQuote() {
+	const char *story = "And I heard a great voice out of the temple saying to the seven angels. "
+						"Go your ways and pour out the vails of the wrath of god upon the earth. "
+						"Book of revelation Chapter 16 verse 7.";
```

Comment:
This is verse 1, not 7.

Also the punctuation is not quite correct. In game the first line ends with a coma: `saying to the seven angels, go your ways`